packages/base/src/sap/ui/webcomponents/base/support/IE11.js

```diff
@@ -0,0 +1,7 @@
+// IE11 only - runtime polyfills for ES5 environments
+import "@ui5/webcomponents-core/dist/sap/ui/thirdparty/es6-promise";
+import "@ui5/webcomponents-core/dist/sap/ui/thirdparty/es6-other-polyfills";
+import "regenerator-runtime/runtime";
+
+// Plus all polyfills neede for edge are also needed for IE11
+import "./Edge";
```

Comment:
This will be imported two times in bundle.esm.js.

The first time from here and the second time from line 1 in the bundle.esm.js file.

Isn't it better to import this line directly in bundle.esm.js? It will be imported both in es5 and esm bundle